tests/acceptance/new-test.js

```diff
@@ -83,8 +83,40 @@ describe('Acceptance: ember new', function() {
       let actualFiles = walkSync('.').sort().filter(e => e !== '.DS_Store');
       expect(actualFiles).to.deep.equal(expectedFiles);
     }));
+
+    it('EMBER_CLI_MODULE_UNIFICATION: ember new adds ember-welcome-page by default', co.wrap(function *() {
+      yield ember([
+        'new',
+        'foo',
+        '--skip-npm',
+        '--skip-bower',
+        '--skip-git',
+      ]);
+
+      expect(file('package.json'))
+        .to.match(/"ember-welcome-page"/);
+
+      expect(file('src/ui/routes/application/template.hbs'))
+        .to.contain("{{welcome-page}}");
+    }));
+
+    it('EMBER_CLI_MODULE_UNIFICATION: ember new --no-welcome skips installation of ember-welcome-page', co.wrap(function *() {
+      yield ember([
+        'new',
+        'foo',
+        '--skip-npm',
+        '--skip-bower',
+        '--skip-git',
+        '--no-welcome',
+      ]);
+
+      expect(file('package.json'))
+        .not.to.match(/"ember-welcome-page"/);
+
+      expect(file('src/ui/routes/application/template.hbs'))
+        .to.contain("Welcome to Ember");
+    }));
```

Comment:
instead of duplicating the test, would it make sense to use the same test and put that last assertion in a `isExperimentEnabled('MODULE_UNIFICATION')` condition?

Reply:
@Turbo87 Yes, it makes much more sense; thank you for the review. Fixed by https://github.com/ember-cli/ember-cli/pull/8195/commits/1519756ad940c714b25b484d69180bb2d223cc85